app/models/email-address.ts

```diff
@@ -0,0 +1,5 @@
+import Model, { attr } from '@ember-data/model';
+
+export default class EmailAddressModel extends Model {
+  @attr('string') declare emailAddress: string;
```

Comment:
```suggestion
  @attr('string') declare value: string;
```

(Same note as in core)